data/json/recipes/other/parts.json

```
@@ -400,26 +400,70 @@
+    "result": "copper_rod",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "10 m",
+    "autolearn": true,
+    "charges": 2,
+    "qualities": [ { "id": "ANVIL", "level": 1 }, { "id": "HAMMER", "level": 2 }, { "id": "CHISEL", "level": 3 } ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_redsmithing" } ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 10 ], [ "oxy_torch", 10 ] ] ],
+    "components": [ [ [ "scrap_copper", 3 ], [ "copper", 80 ] ] ]
+  },
   {
     "type": "recipe",
     "activity_level": "MODERATE_EXERCISE",
+    "result": "copper_wire",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "32 m",
+    "autolearn": true,
+    "charges": 200,
+    "qualities": [ { "id": "ANVIL", "level": 1 }, { "id": "HAMMER", "level": 2 } ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_wiremaking" } ],
+    "//": "Requires annealing 4 times from the copper rod.",
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 40 ], [ "oxy_torch", 40 ] ], [ [ "draw_plate", -1 ] ] ],
+    "components": [ [ [ "copper_rod", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "copper_wire",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "20 m",
+    "autolearn": true,
+    "charges": 200,
+    "qualities": [ { "id": "ANVIL", "level": 1 }, { "id": "HAMMER", "level": 2 } ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" } ],
+    "//": "Requires annealing 4 times from the copper rod.",
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 40 ], [ "oxy_torch", 40 ] ], [ [ "wire_draw_machine", 100 ] ] ],
+    "components": [ [ [ "copper_rod", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
     "result": "cable",
     "category": "CC_ELECTRONIC",
     "subcategory": "CSC_ELECTRONIC_COMPONENTS",
     "skill_used": "fabrication",
     "skills_required": [ "electronics", 2 ],
-    "difficulty": 5,
-    "time": "3 m",
+    "difficulty": 2,
+    "time": "20 s",
     "autolearn": true,
-    "charges": 1,
-    "qualities": [
-      { "id": "ANVIL", "level": 1 },
-      { "id": "HAMMER", "level": 2 },
-      { "id": "CHISEL", "level": 3 },
-      { "id": "CUT", "level": 1 }
-    ],
-    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "surface_heat", 1, "LIST" ], [ "forge", 1 ], [ "oxy_torch", 1 ] ] ],
-    "components": [ [ [ "copper", 1 ] ], [ [ "duct_tape", 1 ] ] ]
+    "charges": 2,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "copper_wire", 2 ] ], [ [ "duct_tape", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "copper_wire", 6 ] ], [ [ "duct_tape", 1 ] ] ]
```
So that a 2g charge of duct tape makes 6 charges of cable, each using 0.33g of duct tape.